docs/quickstart/ech/docker.md

```diff
@@ -14,4 +13,72 @@ nav_exclude: true
 🗄️ Elastic Cloud Hosted
 {: .label .label-blue }
 
-🚧 Coming soon
\ No newline at end of file
+The quick start for Docker with with Elastic Cloud Hosted will guide you through setting up the EDOT Collector and EDOT SDKs to collect host metrics, logs and application traces.
+
+## Instructions
+
+1. Create a `otel-collector-config.yml` file with your EDOT collector configuration. See the [configuration reference](../../_edot-collector/config/default-config-standalone.md) for "Direct ingestion into Elasticsearch" for more details.
+
+2. Retrieve the `Elasticsearch Endpoint` and the `Elastic API Key` for your Elastic Cloud deployment by [following these instructions](./#retrieving-connection-details-for-your-elastic-cloud-deployment).
```

Comment:
Not sure why the link check fails here. This point was copy-pasted from the other guides in the serverless folder.